templates/patterns/card_v2/card_v2.ui_patterns.yml

```diff
@@ -0,0 +1,64 @@
+card_v2:
+  label: 'Card V2'
+  description: 'A card is a flexible and extensible content container. The V2 leaves more possibility for content and render items as we transformed all the content areas into render options. https://v5.getbootstrap.com/docs/5.0/components/card'
+  variants:
+    default:
+      label: 'Default'
+      description: 'Standard bootstrap card v2.'
+  fields:
+    title:
+      type: text
+      label: 'Title'
+      description: 'Card title. Supports markup, e.g. for links.'
+      preview: 'Title card'
+    title_tag:
+      type: text
+      label: 'Title tag'
+      description: 'Card title tag. Default h1.'
+      preview: 'h4'
+    image:
+      type: render
+      label: 'Image'
+      description: 'Image for the card. Optional.'
+      preview: '<img src="https://picsum.photos/1000/500/" alt="Alternative text for card image">'
+    image_position:
```

Comment:
we can remove this, instead I think we can add on off switches for the border and the rounded corners

Reply:
removed.